Firestore/Example/Tests/FIRQueryTests.m

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Please put this file in the location that parallels the implementation: under `API` as a peer of `FIRGeoPointTests.m`.